### PR TITLE
Anti clockwise camera rotation

### PR DESCRIPTION
**Pull request by runelaenen**
Like requested in Issue #1498.
I also changed the way the enter key handles rotations (check the last commit) because for some reason that also stopped working (although it didn't use the window_rotate_camera() function).

### CODE REVIEW

File: src/interface/window.c [hunk 1]
```diff
@@ -1354,7 +1354,7 @@ void sub_688956()
  * 
  *  rct2: 0x0068881A
  */
-void window_rotate_camera(rct_window *w)
+void window_rotate_camera(rct_window *w, int wise)
```
Comment (duncanspumpkin) [inline src/interface/window.c]: i think `direction` would be a better name for this. And add a little comment saying what `direction` is used for.

**Comment by IntelOrca** [comment]
Can you rebase this off develop? as it contains your own merge as the first commit.

**Comment by Gymnasiast** [comment]
Did you use a self-closing dropdown that automatically takes 'clockwise' as its default when just clicking it?

**Comment by IntelOrca** [comment]
@Gymnasiast Yes, you can just click the button to go clockwise.

**Comment by runelaenen** [comment]
Yes I did.
And @IntelOrca this is not what you meant, is it? :s

**Pull request closed without merging**